app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/model/Event.kt

```diff
@@ -34,6 +35,18 @@ data class Event(@PrimaryKey
         const val defaultDuration = 7200000 // Milliseconds
     }
 
+    fun getFormattedDateTime(): String {
+        return "${getFormattedDate()}, ${getFormattedTime()}"
+    }
+
+    fun getFormattedDate(): String {
+        return DateTimeFormat.forPattern("dd MMMM yyyy").print(start)
+    }
+
+    fun getFormattedTime(): String {
+        return DateTimeFormat.forPattern("HH:mm").print(start)
```

Comment:
Same here - how about am/pm here?

Reply:
Yeah right, but the problem of using mediumTime() here is that it will return a time like "2:00 nachm" instead of "14:00", apparently that's some kind of bug. We can instead ask the DateFormat class whether 24 hour format should be used, and then set the pattern accordingly.

You can have a look when I have commited that.